lib/form_data.js

```diff
@@ -211,18 +212,19 @@ FormData.prototype._getContentDisposition = function(value, options) {
 
   var contentDisposition;
 
-  // custom filename takes precedence
+  // custom filepath or filename take precedence
   // fs- and request- streams have path property
   // formidable and the browser add a name property.
-  var filename = options.filename || value.name || value.path;
+  var filename = options.filepath || options.filename || value.name || value.path;
 
   // or try http response
   if (!filename && value.readable && value.hasOwnProperty('httpVersion')) {
     filename = value.client._httpMessage.path;
   }
 
   if (filename) {
-    contentDisposition = 'filename="' + path.basename(filename) + '"';
+    filename = options.filepath ? normalize(filename) : path.basename(filename);
```

Comment:
I see.

Probably we still want to clean up don't, `'./photos/toys/unicycle.jpg'` -> `'photos/toys/unicycle.jpg'`, as for win -> unix, we could do something like:
```
filename = filename.replace('\\', '/');
```